functions/src/modules/SE.functions.ts

```diff
@@ -45,7 +46,7 @@ export const updateNotifyOnNewAllotment = functions.database
         await db.ref(`/sound-editing/tasks/${list}/${taskId}/restoration`).update({
             status: 'Given',
             assignee: allotment.assignee,
-            timestampGiven:Math.round((new Date()).getTime() / 1000),
+            timestampGiven: Math.round((new Date()).getTime() / 1000),
```

Comment:
Can't we get a timestamp of the allotment creation here? Or at least the same pseudo-value for the server-side timestamp as we used somewhere else?

Reply:
okay, done